app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+
+import requests
+
+
+API_KEY = os.getenv("api_key")
+URL = "https://api.weatherapi.com/v1/current.json?"
+FILTERING = "Paris"
```

Comment:
If you set api_key as an environment variable, why not do the same for URL and FILTERING?

Reply:
It's our task condition:
  You need to generate API_KEY for using Weather API, but it shouldn't be hard-coded in main.py.
  You should use environment variables for that purpose. So, read API_KEY from environment. And to pass environment variables to docker container use -e flag.

Reply:
First, in the naming convention, you should use os.getenv("API_KEY"). Second, if you pass API_KEY as an environment variable, ensure it is correctly set. Regarding other fields, I’m confused about why I can’t decide which city I want to see. I want to have a single configuration where I can input the desired city. The same applies to the URL—what happens if it changes? What should we do then?